server/app/service/ses_client.js

```diff
@@ -0,0 +1,50 @@
+const AWS = require('aws-sdk');
+
+const config = require("../../config");
+
+AWS.config.update({
+    accessKeyId: config.aws.key,
+    secretAccessKey: config.aws.secret,
+    region: config.aws.ses.region
+});
+
+const ses = new AWS.SES({apiVersion: '2010-12-01'});
+
+const sendEmail = (to, subject, message) => {
+    const from = 'surekha.gadkari@objectedge.com';
```

Comment:
We need 'from' as parameter to function and default value for from can be read from config

Reply:
The initial implementation that only. Updated.